app/server/Launcher.hs

```diff
@@ -0,0 +1,45 @@
+module Launcher where
+
+import Control.Concurrent.Async
+import Prelude
+import System.Process
+
+-- What is @bracket@?
+--
+-- When you want to acquire a resource, do some work with it, and then release
+-- the resource, it is a good idea to use bracket, because bracket will install
+-- the necessary exception handler to release the resource in the event that an
+-- exception is raised during the computation. If an exception is raised, then
+-- bracket will re-raise the exception (after performing the release).
+--
+-- http://hackage.haskell.org/package/base-4.12.0.0/docs/Control-Exception-Base.html#v:bracket
+
+
+
+
+
+
+
+
+
+-- | Run an external process alongside a haskell IO action
+--
+-- Both are expected to run forever.
+--
+-- Sadly you cannot quit this in ghci
+bracketProcess :: CreateProcess -> IO () -> IO ()
```

Comment:
It's gonna be a bit "impractical" for users of that function to start a process because they'll have to deal with the internals of `CreateProcess` as well. Plus, this starts a process and, runs an IO action in the current process.  In practice, we want the launcher to start both the cardano-http-bridge and the wallet-server. This seemingly means that we need:

- An executable / CLI for the bridge
- An executable / CLI for the wallet

And, run both of them in a separate process, keeping the launcher thread as a "supervisor". So, the API here could be much more like:

```hs
data Command = Command
    { cmdName :: String
    , cmdArgs :: [String]
    }

launch :: [Command] -> IO [ProcessHandle]
```

What we do with the handles and what are the action isn't relevant for now. There's no need to "leak" logic about the node or the bridge within this function. For stdout, there a choice to make. We can either only bind the output of the first command, a bit arbitrarily (and stating it in the function's doc), or bind the output of all the commands, or, leave the decision to the caller and have a flag in the `Command` representation.